packages/flutter/test/material/snack_bar_test.dart

```diff
@@ -1591,7 +1591,7 @@ void main() {
 
           expect(initialBottomLeft, finalBottomLeft);
           expect(initialBottomRight, finalBottomRight);
-        },
+        }, skip: true, // TODO(goderbauer): Figure out test intent, the MediaQuery used in the test was previously ignored.
```

Comment:
Hey @Piinks do you maybe understand what the intent of this test was? Unfortunately, as written the test didn't test what it was claiming. Prior to the change I am making in this PR, the `MaterialApp` in line 1537 introduces its own MediaQuery. That is the MediaQuery visible to the SnackBar/Scaffold. The SnackBar/Scaffold doesn't see the MediaQuery wrapped around the MaterialApp. Therefore, it also doesn't notice any changes to that MediaQuery. However, those changes seem to be important for the test.

When MaterialApp is no longer introducing a MediaQuery (the change in this PR), it sees the changes in MediaQuery and the test is failing for `SnackBarBehavior.floating` on line 1592. The finalBottomLeft ends up being Offset(0.0, 508.0), while the initialBottomLeft was Offset(0.0, 528.0).

Wondering now if the SnackBar implementation has a bug and the test has it right or if the test is bogus. Given that nobody seems to have complained about the current behavior in the last three years since the test was introduced, I am inclined to say that it is the test that's wrong and we should just delete it...

Reply:
I looked at this scenario in dartpad [1] and I am fairly certain now that the test is bogus. The floating SnackBar's bottomLeft is changing positions and that is very much expected. The test was likely only supposed to apply to a fixed SnackBar. I'll change the test accordingly. 

[1] https://dartpad.dev/?id=4dd53931369dcbb9252d4fb8afe8eb36

Reply:
Thanks for investigating. IIRC, the MediaQueryData is heavily manipulated by the Scaffold as it lays out its children, so they all are positioned correctly, so Snackbars do not overlap FABs etc. This test may have been bugged, thanks for catching it!